configs/satellite_config.yaml

```diff
@@ -21,41 +21,42 @@ logger:
   level: "info"
 
 telemetry:
-  namespace: namespaces
+  cluster: namespaces
```

Comment:
What is this renaming for? I also still see `namespace1` in the pipe_name. Could you be clear more?

Reply:
they are different concepts, one is the cluster concept, such that `Namespace` in the Kubernetes. Another one is the internal concept, which means running in the different pipes. Please ignore it now, I would add the env override mod later, which is similar withe the SkyWalking.